pkg/config/config_test.go

```diff
@@ -811,4 +811,73 @@ env=["foo=bar"]`
 		gomega.Expect(config.Containers.BaseHostsFile).To(gomega.Equal("/etc/hosts2"))
 		gomega.Expect(config.Containers.EnableLabeledUsers).To(gomega.BeTrue())
 	})
+	It("ParsePullPolicy", func() {
+		for _, test := range []struct {
+			value  string
+			policy PullPolicy
+			fail   bool
+		}{
+			{
+				value:  "always",
+				policy: PullPolicyAlways,
+			},
+			{
+				value:  "alWays",
+				policy: PullPolicyAlways,
+			},
+			{
+				value:  "ALWAYS",
+				policy: PullPolicyAlways,
+			},
+			{
+				value:  "never",
+				policy: PullPolicyNever,
+			},
+			{
+				value:  "NEVER",
+				policy: PullPolicyNever,
+			},
+			{
+				value:  "newer",
+				policy: PullPolicyNewer,
+			},
+			{
+				value:  "ifnewer",
+				policy: PullPolicyNewer,
+			},
+			{
+				value:  "NEVER",
```

Comment:
```suggestion
				value:  "NEWER",
```
Also needs `IFNEWER`,

Reply:
These are fixed in current PR.